types/office-js-preview/index.d.ts

```diff
@@ -58047,6 +59025,386 @@ declare namespace Excel {
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            percentComplete?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents the task's priority. It is used for the "Priority" history record type.
+                        This is a value between 0 and 10 with 5 being the default priority if not set, where 0 represents the highest priority.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            priority?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents the task's start date, it is used for the "Schedule" history record type.
+                        It is in UTC time zone. Can be set to `null` to remove the start date. It should be set together with `dueDate` to avoid conflicts.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            startDate?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents the task's title. It is used for "Title" history record type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            title?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents task history record's type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            type?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents the TaskHistoryRecord.id property that was undone for the "Undo" history record type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            undoHistoryId?: boolean;
+        }
+        /**
+         *
+         * Represents a task.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        interface TaskLoadOptions {
+            /**
+              Specifying `$all` for the LoadOptions loads all the scalar properties (e.g.: `Range.address`) but not the navigational properties (e.g.: `Range.format.fill.color`).
+             */
+            $all?: boolean;
+            /**
+            *
+            * Gets the comment associated with the task.
+            *
+            * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+            * @beta
+            */
+            comment?: Excel.Interfaces.CommentLoadOptions;
+            /**
+             *
+             * Gets the users to which the task is assigned.
```

Comment:
Maybe. Here and other places
```suggestion
             * Gets the users to whom the task is assigned.
```